lark/lark.py

```diff
@@ -295,6 +316,7 @@ def __init__(self, grammar, **options):
             with FS.open(cache_fn, 'wb') as f:
                 self.save(f)
 
+    # TODO: merge with above
```

Comment:
@erezsh need guidance here. Are you use you want to keep this docstring? If so, it'll show up twice in the docs, which is not optimal

Reply:
You can instead refer to help(lark.lark.LarkOptions) in the docstring of options.

Reply:
LarkOptions is not the interface, it's a helper class. It doesn't really belong in the documentation, except for maybe a short comment.

Reply:
Makes sense. Will change the docs based on this.